zndraw/server/tasks.py

```diff
@@ -615,6 +619,8 @@ def on_disconnect(token: str, sid: str, url: str):
             ses.delete(rmc)
         ses.commit()
 
+    remove_empty_rooms.delay()
```

Comment:
This is a case of a worker spawning another worker (albeit a single one). Is there a way to refactor this logic so that's avoided?

Reply:
replaced it with a chain call